Validation/CTPPS/plugins/CTPPSLHCInfoPlotter.cc

```diff
@@ -83,14 +92,17 @@ void CTPPSLHCInfoPlotter::fillDescriptions(edm::ConfigurationDescriptions &descr
 //----------------------------------------------------------------------------------------------------
 
 void CTPPSLHCInfoPlotter::analyze(const edm::Event &iEvent, const edm::EventSetup &iSetup) {
-  const auto &lhcInfo = iSetup.getData(lhcInfoESToken_);
+  LHCInfoCombined lhcInfoCombined(iSetup, lhcInfoPerLSToken_, lhcInfoPerFillToken_, lhcInfoToken_, useNewLHCInfo_);
 
-  h_beamEnergy_->Fill(lhcInfo.energy());
-  h_xangle_->Fill(lhcInfo.crossingAngle());
-  h_betaStar_->Fill(lhcInfo.betaStar());
-  h2_betaStar_vs_xangle_->Fill(lhcInfo.crossingAngle(), lhcInfo.betaStar());
+  h_beamEnergy_->Fill(lhcInfoCombined.energy);
+  h_xangle_->Fill(lhcInfoCombined.crossingAngle());
+  h_betaStar_->Fill(lhcInfoCombined.betaStarX);  //adjust accordingly to run period
+  // h_betaStar_->Fill(lhcInfoCombined.betaStarY);
```

Comment:
Right. This was changed just to make the associated unit test working. There are several modules in Validation/ which require update of the LHCInfo and it will be done in another PR. Here in particular we should add a 2nd beta* definition: while the crossing angle is only in one direction (only X or Y), with new optics beta_x and beta_y are not anymore the same (this is new and never done till now)